### PR TITLE
docs: update miss Change-Log

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | docs: update miss Change-Log |
| 🇨🇳 Chinese |          - |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at d663691</samp>

This pull request introduces a new feature to the `DatePicker` and `Calendar` components that enables custom date component generation. It updates the English and Chinese changelogs in `CHANGELOG.en-US.md` and `CHANGELOG.zh-CN.md` to document the feature.

### <samp>🤖 Generated by Copilot at d663691</samp>

* Add a new feature to generate custom date components from `DatePicker` and `Calendar` ([link](https://github.com/ant-design/ant-design/pull/42422/files?diff=unified&w=0#diff-e961ad288c43a2feb2d56b696e10c0b670d652dbb17c76c9aa0452d45157977cR23), [link](https://github.com/ant-design/ant-design/pull/42422/files?diff=unified&w=0#diff-1224d5ca5233fdeba67ba2e2dc6fa3e615905253f35dc132c4b4b86f51f9eab4R23))

### CODE REVIEW

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ timeline: true
 `2023-05-15`
 
 - 🔥 New Component ColorPicker. [#41990](https://github.com/ant-design/ant-design/pull/41990) [@RedJue](https://github.com/RedJue)
+- 🔥 DatePicker & Calendar export generate function directly now, no need to import and use by module path. [#41773](https://github.com/ant-design/ant-design/pull/41773)
```
Comment (MadCcc) [inline CHANGELOG.en-US.md:23]:
```suggestion
- 🆕 DatePicker & Calendar export generate function directly now, no need to import and use by module path. [#41773](https://github.com/ant-design/ant-design/pull/41773)
```

File: CHANGELOG.zh-CN.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ timeline: true
 `2023-05-15`
 
 - 🔥 新增颜色选择器组件。[#41990](https://github.com/ant-design/ant-design/pull/41990) [@RedJue](https://github.com/RedJue)
+- 🔥 DatePicker 与 Calendar 组件直接导出生成自定义日期库组件的方法，不再需要通过 module 路径引入使用。[#41773](https://github.com/ant-design/ant-design/pull/41773)
```
Comment (li-jia-nan) [inline CHANGELOG.zh-CN.md:23]:
```suggestion
- 🆕  DatePicker 与 Calendar 组件直接导出生成自定义日期库组件的方法，不再需要通过 module 路径引入使用。[#41773](https://github.com/ant-design/ant-design/pull/41773)
```

**Comment by li-jia-nan** [comment]
@afc163 @MadCcc @zombieJ 求+1

File: CHANGELOG.zh-CN.md [hunk 3]
```diff
@@ -20,6 +20,7 @@ timeline: true
 `2023-05-15`
 
 - 🔥 新增颜色选择器组件。[#41990](https://github.com/ant-design/ant-design/pull/41990) [@RedJue](https://github.com/RedJue)
+- 🆕  DatePicker 与 Calendar 组件直接导出生成自定义日期库组件的方法，不再需要通过 module 路径引入使用。[#41773](https://github.com/ant-design/ant-design/pull/41773)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:23]:
```suggestion
- 🆕 新增 DatePicker.generateCalendar 与 Calendar.generateCalendar 自定义日期库组件方法，不再需要通过路径引入使用。[#41773](https://github.com/ant-design/ant-design/pull/41773)
```

File: CHANGELOG.en-US.md [hunk 4]
```diff
@@ -20,6 +20,7 @@ timeline: true
 `2023-05-15`
 
 - 🔥 New Component ColorPicker. [#41990](https://github.com/ant-design/ant-design/pull/41990) [@RedJue](https://github.com/RedJue)
+- 🆕 DatePicker & Calendar export generate function directly now, no need to import and use by module path. [#41773](https://github.com/ant-design/ant-design/pull/41773)
```
Comment (li-jia-nan) [inline CHANGELOG.en-US.md:23]:
```suggestion
- 🆕 Add DatePicker.generateCalendar & Calendar.generateCalendar custom date component method now, no need to import and use by module path. [#41773](https://github.com/ant-design/ant-design/pull/41773)
```

File: CHANGELOG.en-US.md [hunk 5]
```diff
@@ -20,6 +20,7 @@ timeline: true
 `2023-05-15`
 
 - 🔥 New Component ColorPicker. [#41990](https://github.com/ant-design/ant-design/pull/41990) [@RedJue](https://github.com/RedJue)
+- 🆕 Add DatePicker.generateCalendar & Calendar.generateCalendar custom date component method now, no need to import and use by module path. [#41773](https://github.com/ant-design/ant-design/pull/41773)
```
Comment (afc163) [inline CHANGELOG.en-US.md:23]:
```suggestion
- 🆕 Add DatePicker.generateCalendar & Calendar.generateCalendar custom date component method, no need to import from module path. [#41773](https://github.com/ant-design/ant-design/pull/41773)
```

**Comment by li-jia-nan** [comment]
大佬这两个方法不需要加上反引号吗？

**Review by afc163 — APPROVED** [review]

**Comment by afc163** [comment]
加。

File: CHANGELOG.en-US.md [hunk 6]
```diff
@@ -20,6 +20,7 @@ timeline: true
 `2023-05-15`
 
 - 🔥 New Component ColorPicker. [#41990](https://github.com/ant-design/ant-design/pull/41990) [@RedJue](https://github.com/RedJue)
+- 🆕 Add DatePicker.generateCalendar & Calendar.generateCalendar custom date component method, no need to import from module path. [#41773](https://github.com/ant-design/ant-design/pull/41773)
```
Comment (afc163) [inline CHANGELOG.en-US.md:23]:
```suggestion
- 🆕 Add DatePicker.generateCalendar and Calendar.generateCalendar custom date component method, no need to import from module path. [#41773](https://github.com/ant-design/ant-design/pull/41773)
```

File: CHANGELOG.en-US.md [hunk 7]
```diff
@@ -20,6 +20,7 @@ timeline: true
 `2023-05-15`
 
 - 🔥 New Component ColorPicker. [#41990](https://github.com/ant-design/ant-design/pull/41990) [@RedJue](https://github.com/RedJue)
+- 🆕 Add DatePicker.generateCalendar and Calendar.generateCalendar custom date component method, no need to import from module path. [#41773](https://github.com/ant-design/ant-design/pull/41773)
```
Comment (li-jia-nan) [inline CHANGELOG.en-US.md:23]:
```suggestion
- 🆕 Add `DatePicker.generateCalendar` and `Calendar.generateCalendar` custom date component method, no need to import from module path. [#41773](https://github.com/ant-design/ant-design/pull/41773)
```

File: CHANGELOG.zh-CN.md [hunk 8]
```diff
@@ -20,6 +20,7 @@ timeline: true
 `2023-05-15`
 
 - 🔥 新增颜色选择器组件。[#41990](https://github.com/ant-design/ant-design/pull/41990) [@RedJue](https://github.com/RedJue)
+- 🆕 新增 DatePicker.generateCalendar 与 Calendar.generateCalendar 自定义日期库组件方法，不再需要通过路径引入使用。[#41773](https://github.com/ant-design/ant-design/pull/41773)
```
Comment (li-jia-nan) [inline CHANGELOG.zh-CN.md:23]:
```suggestion
- 🆕 新增 `DatePicker.generateCalendar` 与 `Calendar.generateCalendar` 自定义日期库组件方法，不再需要通过路径引入使用。[#41773](https://github.com/ant-design/ant-design/pull/41773)
```

**Pull request merged**